vite_ruby/vite_ruby.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'dry-cli', '>= 0.7', '< 2'
   s.add_dependency 'rack-proxy', '~> 0.6', '>= 0.6.1'
   s.add_dependency 'zeitwerk', '~> 2.2'
+  s.add_dependency 'logger', '~> 1.6'
```

Comment:
I don’t know if `vite_ruby` actually depends on `~> 1.6` or if it would be happy with an older version of the gem. It looks like `1.6.1` does support Ruby versions back to 2.5.0 so there shouldn’t be any issues there.